builtin/logical/nomad/path_config_access.go

```diff
@@ -13,6 +13,12 @@ const configAccessKey = "config/access"
 func pathConfigAccess(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config/access",
+
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixNomad,
+			OperationSuffix: "access-config",
```

Comment:
why is this reverse the of the path?

Reply:
The generated operationIDs for these are as follows:

nomad-read-access-config
nomad-write-access-config
nomad-delete-access-config
nomad-read-lease-config
nomad-write-lease-config
nomad-delete-lease-config

As you can see, the "config" kind of becomes a noun in this case.

Reply:
Wouldn't it make more sense to have it the other way around per [docs](https://developer.hashicorp.com/vault/api-docs/secret/nomad)? b/c you're configuring access

Reply:
The Configure Access only applies to the "Write/Update" operation. The read operation is: "Read Access Configuration"

https://developer.hashicorp.com/vault/api-docs/secret/nomad#read-access-configuration

Reply:
i like the one-offs better

Reply:
Changed to this:
```
nomad-configure-access
nomad-read-access-config
nomad-delete-access-config
```